src/SamplesApp/SamplesApp.UITests/SampleControlUITestBase.cs

```diff
@@ -263,5 +263,31 @@ internal IAppRect GetScreenDimensions()
 				return _app.GetScreenDimensions();
 			}
 		}
+
+		internal Rectangle GetVisibleBounds()
+		{
+			switch (AppInitializer.GetLocalPlatform())
+			{
+				case Platform.Browser: throw new NotImplementedException(nameof(GetVisibleBounds));
+			}
+
+			string result = null;
+			try
+			{
+				result = _app.InvokeGeneric(nameof(GetVisibleBounds), default)?.ToString();
+				var ltwh = result.Split(',');
+
+				return new Rectangle(
+					(int)double.Parse(ltwh[0]),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5405523)